api/server/server.go

```diff
@@ -1058,14 +1058,11 @@ func (s *Server) postContainersWait(version version.Version, w http.ResponseWrit
 		return fmt.Errorf("Missing parameter")
 	}
 
-	name := vars["name"]
-	cont, err := s.daemon.Get(name)
-	if err != nil {
+	status, err := s.daemon.ContainerWait(vars["name"], -1*time.Second)
+	if err != nil && !strings.Contains(strings.ToLower(err.Error()), "timed out") {
```

Comment:
with a timeout of `-1` would we ever see a "timed out" error?  I'm not sure we would.